watcher/Cargo.toml

```diff
@@ -12,11 +12,16 @@ path = "src/bin/main.rs"
 name = "mc-watcher-db-dump"
 path = "src/bin/db-dump.rs"
 
+[[bin]]
+name = "mc-watcher-avr-export"
```

Comment:
I actually believe this should live with watcher because it makes no sense without a watcherdb, and it should be shipped with the subset of code that watcher nodes operate with.